R/vdiagram-render.R

```diff
@@ -67,7 +67,10 @@ createVdiagram <- function(data) {
 
   vplot <- testCallouts(vplot, xlimits = xlim(vplot)$side.1)
   
-  ylims <- ylim(vplot)$side.2
+  ylims <- c(
+      min(c(ylim(vplot)$side.2, vdiagramData$minStage)),
+      max(c(ylim(vplot)$side.2, vdiagramData$maxStage))
+      )
```

Comment:
whoops - that would be since we still aren't changing the limits. You should be able to just add `vplot <- view(vplot, side=2, ylim=ylims)`